[2022-04-03 21:03:20,170] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_3 2022-04-03T21:03:14.385109+00:00 [queued]>
[2022-04-03 21:03:20,192] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_3 2022-04-03T21:03:14.385109+00:00 [queued]>
[2022-04-03 21:03:20,193] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-04-03 21:03:20,193] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-04-03 21:03:20,193] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-04-03 21:03:20,226] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_3> on 2022-04-03T21:03:14.385109+00:00
[2022-04-03 21:03:20,228] {standard_task_runner.py:52} INFO - Started process 11177 to run task
[2022-04-03 21:03:20,235] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_3', '2022-04-03T21:03:14.385109+00:00', '--job-id', '51', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmp9prur41o', '--error-file', '/tmp/tmp1xkpvy7b']
[2022-04-03 21:03:20,241] {standard_task_runner.py:77} INFO - Job 51: Subtask task_3
[2022-04-03 21:03:20,335] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_3 2022-04-03T21:03:14.385109+00:00 [running]> on host airflowvm
[2022-04-03 21:03:20,502] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2022-04-03T21:03:14.385109+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-03T21:03:14.385109+00:00
[2022-04-03 21:03:20,502] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-04-03 21:03:20,503] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-04-03 21:03:20,511] {subprocess.py:75} INFO - Output:
[2022-04-03 21:03:23,534] {subprocess.py:83} INFO - Command exited with return code 0
[2022-04-03 21:03:23,598] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_3, execution_date=20220403T210314, start_date=20220403T210320, end_date=20220403T210323
[2022-04-03 21:03:23,690] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-03 21:03:23,706] {local_task_job.py:151} INFO - Task exited with return code 0
